cscs-checks/microbenchmarks/gpu_burn/gpu_burn_test.py

```diff
@@ -44,21 +44,29 @@ def __init__(self):
         self.perf_patterns = {
             'perf': sn.min(sn.extractall(
                 r'GPU\s+\d+\(\S*\): (?P<perf>\S*) GF\/s', self.stdout,
-                'perf', float))
+                'perf', float)),
+            'max_temp': sn.max(sn.extractall(
+                r'GPU\s+\d+\(\S*\): (?P<perf>\S*) GF\/s  \
```

Comment:
You can create a variable for the pattern and use it in both the `perf` and `max_temp`.

Reply:
Done